src/devices/src/virtio/mod.rs

```diff
@@ -331,11 +336,24 @@ pub(crate) mod tests {
                 kernel_cmdline: &mut self.kernel_cmdline,
             }
         }
+        #[cfg(target_arch = "aarch64")]
+        // Support only GIC_V3.
```

Comment:
Let's add more comments here: adding a dummy GIC V3 so that we can register the IRQ fds and thus test the virtio functionality in arm as well.